src/stactools/cop_dem/assets_utils.py

```diff
@@ -0,0 +1,18 @@
+"""
+Creates the Item Assets section
+"""
+import os.path
+from typing import Tuple
+
+
+def change_asset_directory(href: str, asset_name: str,
+                           subdirectory: str) -> Tuple[str, str]:
```

Comment:
Since all 3 parameters are strings, it can be easy to specify arguments in the wrong order without being alerted to the problem. To help reduce the chances of this, you might want to consider kwargs-only parameters:
```suggestion
def change_asset_directory(*, href: str, asset_name: str,
                           subdirectory: str) -> Tuple[str, str]:
```

Then, when calling the function, the caller is forced to specify named arguments. While it would still be possible to mis-specify arguments, you at least stand a better chance of not doing so because you'll have an immediate visual clue that something is wrong).

```
# Not obvious that args 2 and 3 are swapped, and might
# even seem like a more logical argument order
change_asset_directory(href, subdir, asset)

# Order doesn't matter, and names make it less likely to
# supply the wrong values with the names
change_asset_directory(href=href, subdir=subdir, asset=asset)
```